README.md

```diff
@@ -15,30 +14,7 @@
    +  Create documents
    +  Delete documents, users
    +  Update documents, users
-
-## Endpoints
```

Comment:
I agree, but the feedback a friend got during his defense said to remove it. That was why I removed it.